frontend/packages/dev-console/src/components/dropdown/ResourceDropdown.tsx

```diff
@@ -137,6 +140,10 @@ class ResourceDropdown extends React.Component<ResourceDropdownProps, State> {
           : _.get(_.keys(sortedList), 0);
     }
     selectedItem && this.handleChange(selectedItem, sortedList);
+
+    if (nextProps.loaded && !this.props.loaded && _.isEmpty(sortedList) && onLoad) {
```

Comment:
You don't need to check if the list is empty here. Always notify.

Reply:
Yes, no need to check for an empty list as now I am sending the list item.

Reply:
Done. Remove the check for the empty list.